nix_update/version/github.py

```diff
@@ -33,7 +35,11 @@ def fetch_github_version(url: ParseResult, version_regex: str) -> Optional[str]:
     releases = tree.findall(".//{http://www.w3.org/2005/Atom}entry")
     entries = [version_from_entry(x) for x in releases]
     filtered = [
-        x for x in entries if x is not None and extract_version(x, version_regex)
+        x
+        for x in entries
+        if x is not None
+        and extract_version(x, version_regex)
+        and ((not stable_version) or version_is_stable(x))
```

Comment:
I would actually keep the latest unstable version here. If no stable versions is found an error message could be printed that an unstable version could be used if the flag is given.